app/src/components/ui/Radius.vue

```diff
@@ -24,31 +24,48 @@
             },
         },
         data(): {
-            selectedRadius: string,
+            newSelectedRadius: string,
+            radii: Array<{text: string, value: string}>
```

Comment:
is this enough of a recurring concept for us to offer a `SearchRadius` type ?

Reply:
✓